# Evidence for activation of nitenpyram by a mitochondrial cytochrome P450 in *Drosophila melanogaster*

Thomas W.R. Harrop^a,b,1^,  Shane Denecke^a,1^, Janice Chan^a^, Tinna Yang^a^, Phillip J. Daborn^a^, Trent Perry^a^, and Philip Batterham^a,2^

^a^ Department of Genetics and Bio21 Molecular Science and Biotechnology Institute, The University of Melbourne, Victoria 3010, Australia.

^b^ Present address: Department of Biochemistry, The Laboratory for Evolution and Development, The University of Otago, Dunedin 9054, Aotearoa-New Zealand.

^1^ These authors contributed equally to this work

^2^ Corresponding author: p.batterham@unimelb.edu.au

### Keywords

### Abstract 

### Graphical abstract 

### Highlights

## Introduction

Cytochrome P450 genes (P450s) are found in all eukaryotic organisms and are involved in diverse physiological processes. Some insect P450s have essential developmental roles in hormone synthesis and degradation, while others act as drug metabolizing enzymes (DMEs **Where does this term come from? Can we call them xenobiotic-metabolising enzymes instead?**) capable of modifying the chemical structure of insecticides from different classes and affecting resistance. Increased expression of P450s is often positively associated with resistance, particularly to the neonicotinoid class of insecticides (Bass et. al 2015). Individual P450s have been associated with neonicotinoid resistance by comparing transcript levels between resistant and susceptible strains or by genetic mapping (*e.g.* Daborn 2002, Karunker 2008, Puinean 2010, Bass 2011, Bass 2013, Ding 2013, Yang et. al 2013 **any examples of mapping neonicitonoid resistance to P450s, other than *Cyp6g1*?**), and some candidates have been confirmed by transgenic overexpression in *Drosophila melanogaster* (Daborn et. al 2002, Zhu et. al 2010, Daborn et. al 2012, Bass et. al 2013).

**Should we include a sentence or two about nitenpyram and its metabolism? There are some refs in Tomizawa (DOI: 10.1146/annurev.pharmtox.45.120403.095930) but I didn't really understand it and I couldn't get any full text**. CYP6G1 metabolites of the neonicotinoid imidacloprid have been identified and characterised *in vitro* and *in vivo* (Joußen et. al 2008, Hoi et. al 2014). Some of these metabolites may be more toxic than the parent insecticide. For example, the olefin metabolite of imidacloprid has a lower LC~50~ than imidacloprid and higher affinity for the neonicotinoid target protein, nAChR (Nauen et. al 1998), but is more **rapidly?** excreted from *D. melanogaster* larvae (Fusetto et. al 2017 **could you send me this paper?**). These studies have challenged the earlier paradigm that P450-mediated metabolism always leads to detoxification.

P450 genes form four phylogenetic clades, three encoding proteins localised to the endoplasmic reticulum and one encoding enzymes localised to mitochondria (Ranson et. al 2002, Feyereisen 2006). Although all vertebrate mitochondrial P450s appear to have developmental functions (Guengerich et. al 2005), arthropod mitochondrial P450s can function in development or as DMEs (Gilbert et. al 2004, Daborn et. al 2007). Here, we report evidence that *Cyp12a5*, a mitochondrial P450, is involved in the *in vivo* metabolism of nitenpyram in *D. melanogaster*, possibly forming a metabolite with higher toxicity than nitenpyram.

## Material and methods 

### Transgenic fly lines

To generate *Cpr* and *dare* RNAi lines, we amplified two non-overlapping RNAi fragments from each gene using the Expand High Fidelity Plus PCR System (Roche Applied Science), cloned them into pGEM-T Easy (Promega) and subcloned them twice in separate steps into the *Nhe*I and *Avr*II sites of pWIZ (Lee and Carthew 2003) so that the fragment was in the opposite direction in each site (**supporting figure 1**). The primers are listed in **supporting table 2**. The recombinant pWIZ plasmids were transformed into the *w*^1118^ strain of *D. melanogaster* and transgenic progeny with autosomal insertions were selected for further characterisation.

We constructed *Cyp12a4* and *Cyp12a5* UAS lines by amplifying the open reading frames from the *y* ; *cn bw sp* reference strain of *D. melanogaster* using Q5 High-Fidelity DNA Polymerase (New England BioLabs), cloning them into pGEM-T Easy and subcloning them into the pUASt-attB vector via the *Not*I sites. We injected the recombinant plasmids into a target strain of genotype *y*^1^ M{vas-int.Dm}ZH-2A *w*^\*^ ; P{CaryP}attP40 (hereinafter attP40 background), carrying the X-chromosome from Bloomington stock 24482 and the autosomes from Bloomington stock 25709 **is this correct?**. We used  strains from the Vienna Drosophila Resource Centre KK library for RNAi knockdown of *Cyp12a4* and *Cyp12a5* (VDRC stock numbers 107088 and 103961, respectively). 


### *Cpr* and *dare* RNAi phenotyping 

We tested the lethality of ubiquitous RNAi knockdown of *Cpr* and *dare* by crossing male RNAi lines to virgin female 5´*tubulin* (genotype *y*^1^ *w*^\*^ ; P{tubP-GAL4} / TM3, *Sb*^1^, Bloomington stock number 5138) and 5´*Actin* (genotype *y*^1^ *w*^\*^; P{Act5C-GAL4} / TM6B, *Tb*^1^, Bloomington stock number 3954) driver lines. We repeated the crosses at 22°C and 29°C and counted emergence of progeny after 15 days. To determine the life stage mortality of flies expressing the *Cpr* and *dare* RNAi constructs in the ring gland, we crossed the RNAi lines to a 5´*phantom* driver (Guittard et al 2011), using a cross between the driver and *w*^1118^ to control genetic background. We assayed mortality by placing three replicates of 20 eggs from each cross in normal Drosophila food vials. The driver line included a balancer chromosome that constitutively expresses GFP (*CyO*-GFP), and progeny that inherit the balancer chromosome do not produce GAL4 and were expected to be viable. Larvae expressing GAL4 were collected at first instar stage by separation using a SZX12 fluorescence stereomicroscope (Olympus). Emerging adults were scored as *Cy*^1^ (control) or *Cy*^+^ (RNAi). We determined life stage mortality by placing first instar larvae of each genotype indivudally (*n* = 40) into 5 mL vials containing 1 mL of normal Drosophila medium. We supplied pulses of 8.7 μg of 20-hydroxyecdysone in a yeast paste wetted with 10 % v/v ethanol after two and six days to half of the larvae. The remaining larvae were treated with yeast paste wetted with ethanol without 20-hydroxyecdysone. We inspected vials daily to determine life stage and mortality. To measure the lethality of *Cpr* and *dare* RNAi in the *D. melanogaster* digestive system, we crossed male RNAi lines to virgin females with the 5´*Cyp6g1*HR construct, which express GAL4 in the midgut, Malpighian tubules and fat body (Chung et al 2007), over a balancer chromosome. We inferred lethality by scoring the ratio of adults emerging with the balancer phenotype (*Antp*^Hu^).

### Insecticide bioassays

The homozygous 5´*Cyp6g1*HR-3a driver line (Chung et al 2007) was used for all bioassays, and crosses with the recipient strain of the *Cpr* and *dare* or *Cyp12a4* and *Cyp12a5* RNAi constructs (*w*^1118^ and Bloomington stock 60100, respectively) or the attP40 background were used to control genetic background. We exposed replicates of 20 4-day-old, mated, adult female progeny to discriminating doses of 2, 4 or 8 μg·vial^−1^ of DDT (Sigma) using 24 hour contact assays in glass scintillation vials (Daborn et al., 2001). We reared replicates of 25 or 50 first instar larvae on food containing discriminating doses of 1 or 1.5 × 10^−4^ % w/v nitenpyram (Novartis) and 0.9 or 1.1 × 10^−6^ w/v dicyclanil (Novartis) and counted emergence after 15 days. The larval assays for the RNAi constructs were were performed on semolina medium, while corn meal medium was used in the *Cyp12a4* and *Cyp12a5* overexpression bioassays **do we have refs for normal/hoff food?**. We corrected the mortality data for control mortality using Abbott’s formula (Hoekstra, 1987), and calculated confidence intervals that take into account the error in both the control and experimental mortality using the formulae described by Rosenheim and Hoy (1989).

### Overexpression and RNAi quantification

To quantify the knockdown of *Cpr* and *dare*, we collected three biological replicates of five midguts from first and third instar larvae and four-day-old adults. We homogonized tissues in TRIzol Reagent (Invitrogen) at 20.0 Hz for 4 minutes in a Mixer Mill MM 400 (Retsch), and isolated RNA from the homogenate by chloroform extraction and isopropanol precipitation. After testing RNA quality and yield by agarose gel electrophoresis and a fluorometric assay (Qubit Fluorometer, Invitrogen), we generated cDNA with the SuperScript III First-Strand Synthesis System (Invitrogen) using random hexamer oligonucleotides. We validated the primers for use with the $e^{- \Delta \Delta C_{T}}$ method (Livak and Schmittgen, 2001; Vandesompele et al., 2002), and used SYBR Green Master Mix (Applied Biosciences) on a RotorGene RG-3000 (Corbett) for quantitative real-time RT-PCR. We quantified *Cyp12a4* and *Cyp12a5* overexpression with the same method, except that we extracted RNA with TRISure reagent (Bioline) and generated cDNA with GoScript reverse transcriptase (Promega). Primers used for real-time PCR are listed in **supporting table 2**.

## Results

### RNAi of *dare*, a mitochondrial P450 redox partner, reduces nitenpyram mortality

To detect phenotypes arising from disruption of cytochrome P450 functions in specific tissues, we generated an RNAi system to interrupt the functions of two P450 redox partners, *Cpr* and *dare*, under control of the GAL4-UAS system. We tested two non-overlapping UAS-RNAi constructs for each redox partner (**supporting figure 1**). Ubiquitous RNAi with the 5´*tubulin* driver or specific RNAi in the ring gland using the 5´*phantom* driver resulted in lethality, and the latter was partially rescued by addition of 20-hydroxyecdysone (**supporting figure 2**), suggesting that RNAi of the redox partners inhibits P450 functions.

We then used the redox partner RNAi system to examine phenotypes related to P450-based metabolism of insecticides in the insect digestive system. We crossed the redox partner RNAi lines to the 5´*Cyp6g1*HR-3a driver, which produces GAL4 in the midgut, Malpighian tubules and fat body. We detected 60–80% removal of *Cpr* mRNA and 70–90% of *dare* mRNA in midguts from first instar larval, third instar larval and adult progeny by quantitative real-time RT-PCR (table 1). To assess the lethality of the redox partner RNAi system in these tissues, we crossed heterozygous flies carrying the driver transgene and a balancer chromosome (genotype 5´*Cyp6g1*HR-3a / *TM6B*, *Tb*^1^) to the *Cpr* and *dare* RNAi lines. There was no reduction in emergence of *Antp*^+^ progeny, suggesting that RNAi of the P450 redox partners in these tissues does not cause developmental lethality.
    
To assess the effect of *Cpr* and *dare* knockdown in the *D. melanogaster* metabolic tissues, we reared flies expressing the UAS-RNAi constructs under control of the 5´*Cyp6g1*HR-3a driver on nitenpyram or dicyclanil or exposed them to DDT as adults. Mortality of flies expressing either *dare* RNAi construct reared on two different concentrations of nitenpyram was higher compared to control crosses (Figure 1). This suggests that metabolism of nitenpyram by one or more mitochondrial P450s expressed in the midgut, Malpighian tubules or fat body leads to the formation of a more toxic product.

### Modulation of *Cyp12a5* expression level affects nitenpyram mortality

To test the role of individual mitochondrial P450s in nitenpyram resistance, we reared first instar larvae expressing UAS-RNAi constructs against *Cyp12a4* and *Cyp12a5* under control of the 5´*Cyp6g1*HR-3a driver on nitenpyram medium. Mortality was lower than the control cross for *Cyp12a5* flies, but not for *Cyp12a4* flies (Figure 1). This suggests that *Cyp12a5* is a candidate for activation of nitenpyram in the *D. melanogaster* metabolic tissues. **No qPCR for this. Tinna and Janice did the nitenpyram assays after I left. I expect we'd have to dissect midguts to detect any change, like for *Cpr* and *dare***.

To test the effect of constitutive overexpression of *Cyp12a4* and *Cyp12a5*, we crossed transgenic UAS lines containing the open reading frame from each gene (UAS-*Cyp12a4* and UAS-*Cyp12a5*) to the 5´*Cyp6g1*HR-3a driver and reared first instar larval progeny on nitenpyram. Flies overexpressing *Cyp12a5* had increased mortality compared to the control cross, whereas no change in mortality was detected for *Cyp12a4* (Figure 1). **qPCR for o/e**. These results support the negative correlation between *Cyp12a5* expression and nitenpyram resistance.

Because resistance to lufenuron maps to the *Cyp12a4*/*Cyp12a5* region (Bogwitz 2005), we also reared first instar larvae overexpressing *Cyp12a4* or *Cyp12a5* on lufenuron. Mortality of UAS-*Cyp12a4* progeny decreased mortality on lufenuron compared to the control cross, but no change was observed for UAS-*Cyp12a5* progeny (**supporting figure X**).
    
## Discussion

- Bring the discussion back to o/e of P450s as the major mechanism of neonicotinoid resistance, Cyp12a5 is a gene with "resistance potential" 
- Get housefly *CYP12A1* in there (mine from feyereisen 2011); likewise *Cyp12a4* and *Cyp12d1* (both Dros)

a combination of RNAi and GAL4-UAS overexpression is used to characterize the roles of a subset of P450s and their redox partners in vivo using D. melanogaster. RNAi knockdown of the P450 reductases Cpr and dare was lethal if performed ubiquitously or in the ring gland. Partial rescue with ecdysone suggested that this was due to the essential role Cpr and dare play in facilitating essential P450 function. Conversely, Cpr and dare knockdown in the digestive tissues (malpighian tubules, midgut and fat body) had no fitness cost but, inhibition of dare led to an increase in nitenpyram resistance. Investigation of a subset of mitochondrial P450s showed that RNAi knockdown of Cyp12a5 phenocopied the dare RNAi (increased resistance) and that GAL4/UAS overexpression of the same gene lead to a decrease in nitenpyram resistance. These results suggest that Cyp12a5 may play a role in increasing the toxicity of nitenpyram.

Test citation [@daborn_single_2002]. 

## Acknowledgements

**Add grant details**.

## References

<div id="refs"></div>
